components/MapChart.js

```diff
@@ -131,6 +175,7 @@ class MapChart extends React.Component {
             )
           )}
         </ComposableMap>
+        <DetailView datMarker={marker} />  
```

Comment:
putting the if check here would make it so that nothing renders (so no div) if `marker` is null:

```javascript
{marker && <DetailView datMarker={marker} />}
```

Then DetailView above could be just the content we want

Reply:
When I tried that, I got a React error that I didn't understand... and then Nikhil tried and the same thing happened — not sure why but maybe you could try it and then explain the error? ;)